plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationIndicatorLayerRenderer.kt

```diff
@@ -92,6 +93,11 @@ internal class LocationIndicatorLayerRenderer(
     style?.removeStyleImage(SHADOW_ICON)
   }
 
+  override fun updateStyle(style: StyleInterface) {
+    this.style = style
```

Comment:
can we compare with previous version?

Reply:
we have basically a lot of occurrences in this PR but all of them are coming from a single place and it's basically `StyleObserver` where we create the new object everytime style is loaded. to sum up I see no sense in adding those checks, imo it simply makes code a bit more polluted

Reply:
oh, sorry
didn't notice that it happened inside internal class. thought that user can set this style.

Reply:
no, all the work done in this PR is not exposed to an end user